README.adoc

```diff
@@ -1,71 +1,6 @@
 = FreeBSD Quarterly Status Reports
 
-== Drafts
+This repository has been used to collect and review FreeBSD quarterly status reports from 2017Q4 through 2022Q4.
+It is now archived.
 
-This repository is for drafts.
-
-Please submit your draft **as early as possible during the quarter**.
-
-Don't worry excessively about language or link:https://docs.asciidoctor.org/asciidoc/latest/syntax-quick-reference/[AsciiDoc syntax] – we can correct things for you.
-
-To submit:
-
-1. Fork the repository. It's OK to reuse an existing fork.
-2. Create a new branch. 
-3. Enter the directory for the current period – `2022q4`.
-4. Create a new file – use the `report-sample.adoc` template. Choose a filename that signifies the name of your project.
-5. Submit a pull request. 
-If you allow edits from maintainers, this can streamline things. 
-6. Someone will review your pull request.
-7. If changes are requested: 
-** evaluate them
-** make any that you think are warranted
-** explain any changes that you declined
-** push again – but please, *do not squash or force-push* (doing so will make the next review much more difficult).
-8. Once your draft is accepted and merged, you're good.
-
-You can tweak things after a merge: simply edit, then submit another pull request.
-
-Thanks!
-
-=== Reviewing
-
-Minor reviews may occur at any time. 
-
-Pre-final reviews occur during the third month of the quarter. 
-
-=== Deadlines for submission
-
-Generally: 
-
-* 31st of March
-* 30th of June
-* 30th of September
-* 31st of December.
-
-For portmgr@, one week later: 
-
-* 8th of January
-* 8th of April
-* 8th of July
-* 8th of October.
-
-==== General deadline for `2022q4`
-
-* 31st of December 2022.
-
-==== Lateness
-
-Send an e-mail to quarterly-submissions@ to request an extension.
-
-== Finals
-
-Collated reports are:
-
-* Sent to link:https://lists.freebsd.org/subscription/freebsd-announce[freebsd-announce]. 
-* Listed at link:https://www.freebsd.org/news/status/[FreeBSD Status Reports].
-* Promoted through link:https://www.freebsd.org/news/newsflash/[FreeBSD News Flash].
-
-The most recent report:
-
-* https://www.freebsd.org/status/report-2022-07-2022-09/
+Please check https://www.FreeBSD.org/status/ for up to date information about FreeBSD status reports collection, reviewing and publication.
```

Comment:
I'd make the URL a link, perhaps `Please check for link:https://www.FreeBSD.org/status/[up to date information about FreeBSD status reports collection, reviewing and publication].`

Reply:
Good idea. How about `Please check the link:https://www.FreeBSD.org/status/[FreeBSD status reports webpage] for up to date information about status reports collection, reviewing and publication.`?

Reply:
> Good idea. How about `Please check the link:https://www.FreeBSD.org/status/[FreeBSD status reports webpage] for up to date information about status reports collection, reviewing and publication.`?

Even better.